qutebrowser/browser/downloads.py

```diff
@@ -364,6 +370,16 @@ def cancel(self, remove_data=True):
         self.finished.emit()
         self.data_changed.emit()
 
+    def path_suggestion(self):
+        """Get the suggestion of file path"""
+        suggestion = config.get('completion', 'download-path-suggestion')
+        if suggestion == 'path':
+            return self._download_dir()
+        elif suggestion == 'filename':
+            return self.basename
+        else:
```

Comment:
I'd prefer to do this this way so it fails loudly if `config.get` returns something unexpected:

```python
...
elif sugestion == 'both':
    ...
else:
    raise ValueError("Invalid suggestion value {}!".format(suggestion))
```